src/extension.ts

```diff
@@ -312,6 +312,7 @@ export function activate(context: vscode.ExtensionContext): void {
             vscode.window.showInformationMessage(`File logging has already been enabled for ${siteTreeItem.client.fullName}.`);
         }
     });
+    registerCommand('appService.OpenCosmosDBExtension', () => vscode.commands.executeCommand('extension.open', 'ms-azuretools.vscode-cosmosdb'));
```

Comment:
This works without editing the package.json? I feel like you need to add the `OpenCosmosDBExtension` command to package.json...

Reply:
It works, will add it to package